cordova-plugin-local-notification/cordova-plugin-local-notification.d.ts

```diff
@@ -0,0 +1,177 @@
+// Type definitions for Apache Local Push Notification plugin
+// Project: https://github.com/katzer/cordova-plugin-local-notifications
+// Definitions by: Paul Thiel <https://github.com/Lordnoname>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * The plugin provides executing local push notifications on the device
+ */
+interface CordovaPluginLocalNotification {
```

Comment:
did you use this pattern? http://definitelytyped.org/guides/best-practices.html#ghost-modules